packages/mdc-slider/README.md

```diff
@@ -0,0 +1,166 @@
+  </li>
+</ul>
+
+## Installation
+
+```
+npm install @material/slider
+```
+
+## Basic Usage
+
+### HTML Structure
+
+```html
+<div class="mdc-slider" tabindex="0" role="slider"
+     aria-valuemin="0" aria-valuemax="100" aria-valuenow="0"
+     aria-label="Select Value">
+  <div class="mdc-slider__track">
+    <div class="mdc-slider__track-fill"></div>
+  </div>
+  <div class="mdc-slider__thumb">
+    <svg class="mdc-slider__thumb-handle" width="24" height="24">
+      <circle cx="12" cy="12" r="6"></circle>
+    </svg>
+  </div>
+</div>
+```
+
+### Styles
+
+```scss
+@import "@material/slider/mdc-slider";
+```
+
+### JavaScript Instantiation
+
+```js
+import {MDCSlider} from '@material/slider';
+
+const slider = new MDCSlider(document.querySelector('.mdc-slider'));
+```
+
+## Variants
+
+### Continuous Slider
+
+Use continuous sliders allow users to make meaningful selections that don’t require a specific value. Continuous Slider is the default variant.
+
+### Initializing the slider with custom ranges/values
+
+When `MDCSlider` is initialized, it reads the element's `aria-valuemin`, `aria-valuemax`, and
+`aria-valuenow` values if present and uses them to set the component's `min`, `max`, and `value`
+properties. This means you can use these attributes to set these values for the slider within the
+DOM.
+
+```html
+<div class="mdc-slider" tabindex="0" role="slider"
+     aria-valuemin="-5" aria-valuemax="50" aria-valuenow="10"
+     aria-label="Select Value">
+  <!-- ... -->
+</div>
+```
+
+## Style Customization
+
+### CSS Classes
+
+CSS Class | Description
+--- | ---
+`mdc-slider` | Mandatory.
+`mdc-slider__track` | Mandatory. Element containing the track-fill.
+`mdc-slider__track-fill` | Mandatory. The fill element to display where the value is.
+`mdc-slider__thumb` | Mandatory. Element containing the thumb-handle.
+`mdc-slider__thumb-handle` | Mandatory. The handle element to display where the value is.
+
+## `MDCSlider` Properties and Methods
+
+| Property | Value Type | Description |
+| --- | --- | --- |
+| `value` | `number` | The current value of the slider. Changing this will update the slider's value. |
+| `min` | `number` | The minimum value a slider can have. Values set programmatically will be clamped to this minimum value. Changing this property will update the slider's value if it is lower than the new minimum |
+| `max` | `number` | The maximum value a slider can have. Values set programmatically will be clamped to this maximum value. Changing this property will update the slider's value if it is greater than the new maximum |
+
+| Method Signature | Description |
+| --- | --- |
+| `layout() => void` | Recomputes the dimensions and re-lays out the component. This should be called if the dimensions of the slider itself or any of its parent elements change programmatically (it is called automatically on resize). |
+
+### Additional Information
+
+#### Events
```

Comment:
nit: This extra `Events` heading can be omitted

Reply:
+1